spark/src/test/java/org/apache/iceberg/spark/data/TestSparkOrcReader.java

```diff
@@ -60,5 +61,29 @@ protected void writeAndValidate(Schema schema) throws IOException {
       }
       Assert.assertFalse("Should not have extra rows", actualRows.hasNext());
     }
+
+    // Also write and validate where all rows are the same (repeating)
+    final Iterable<InternalRow> expectedRepeating = Collections.nCopies(100, expected.iterator().next());
```

Comment:
Let's add new test cases instead of appending new validations to old ones.

Reply:
Fixed